server/apps/entry/schema.py

```diff
@@ -68,18 +68,23 @@ class Meta:
 class EntryType(DjangoObjectType):
     class Meta:
         model = Entry
-        exclude_fields = ('reviewers',)
+        exclude_fields = ('reviews',)
 
     created_by = graphene.Field('apps.users.schema.UserType')
     last_modified_by = graphene.Field('apps.users.schema.UserType')
     figures = DjangoPaginatedListObjectField(FigureListType,
                                              pagination=PageGraphqlPagination(
                                                  page_size_query_param='perPage'
                                              ))
+    latest_reviews = graphene.List('apps.review.schema.ReviewType')
     reviewers = graphene.Dynamic(lambda: DjangoPaginatedListObjectField(
         get_type('apps.users.schema.UserListType'),
         accessor='reviewers'
     ))
+    review_comments = graphene.Dynamic(lambda: DjangoPaginatedListObjectField(
+        get_type('apps.review.schema.ReviewCommentListType'),
+        accessor='review_comments'
+    ))
```

Comment:
Can we use this format? It's hard to get if `get_type(..` is argument for Dynamic or DjangoPag...
```suggestion
    review_comments = graphene.Dynamic(
        lambda: DjangoPaginatedListObjectField(
            get_type('apps.review.schema.ReviewCommentListType'),
            accessor='review_comments'
        )
    )
```